api/src/main/java/org/pmiops/workbench/cohortbuilder/querybuilder/CodesQueryBuilder.java

```diff
@@ -46,21 +46,18 @@
      * https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#in-operators
      */
     private static final String INNER_SQL_TEMPLATE =
-            "select distinct person_id " +
-                    "from `${projectId}.${dataSetId}.${tableName}` a, `${projectId}.${dataSetId}.concept` b "+
-                    "where a.${tableId} = b.concept_id "+
-                    "and b.vocabulary_id in (@cm,@proc) " +
-                    "and b.concept_code in unnest(${conceptCodes})";
+            "select distinct person_id\n" +
+                    "from `${projectId}.${dataSetId}.${tableName}` a, `${projectId}.${dataSetId}.concept` b\n"+
+                    "where a.${tableId} = b.concept_id\n"+
+                    "and b.vocabulary_id in (@cm,@proc)\n" +
+                    "and b.concept_code in unnest(${conceptCodes})\n";
```

Comment:
So if you pass in an array as a query parameter, you use UNNEST?

Reply:
[You _must_ use unnest](https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#in-operators) with an array.  With a subquery you can just use `IN`.